onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py

```diff
@@ -13,26 +15,134 @@
 import numpy.typing as npt
 import onnx
 from onnx.onnx_pb import GraphProto, ModelProto, NodeProto, TensorProto
+from packaging import version
 
 from onnxruntime.capi._pybind_state import quantize_matmul_4bits
 
+from .calibrate import CalibrationDataReader
 from .onnx_model import ONNXModel
 from .quant_utils import attribute_to_kwarg
 
 logging.basicConfig(format="%(asctime)s %(name)s [%(levelname)s] - %(message)s", level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
+class WeightOnlyQuantConfig:
+    def __init__(
+        self,
+        algorithm,
+        model_path,
```

Comment:
this model_path seems duplicated with MatMul4BitsQuantizer.model. Could we only keep one of them?

Reply:
Hi, we removed `model_path`. Please help review. Thanks!